src/Themes/Button.css

```
@@ -0,0 +1,6 @@
+.Button{
+    background-color: yellow;
+    width: 60px;
+    height: 30px;
+    border: none;
```

We need a `border`. Maybe set to
```suggestion
    border: 1px solid {same-as-background-color};
```